tests/models/test_layers_utils.py

```diff
@@ -314,59 +314,6 @@ def test_restnet_with_kernel_sde_vp(self):
         assert torch.allclose(output_slice.flatten(), expected_slice, atol=1e-3)
 
 
-class AttentionBlockTests(unittest.TestCase):
```

Comment:
I re-configured these tests with the equivalent configuration on `Attention` and manually ran them on test target devices to confirm they still pass. We can just remove the tests

Reply:
Cool!